buildconfig/vstools.py

```diff
@@ -1,5 +1,3 @@
-# -*- coding:latin-1 -*-
```

Comment:
If this file is latin-1 encoded then a encoding conversion should be done! (not sure if it would be visible in the diff)

Reply:
Are there any non-ASCII characters in this file?

Reply:
I don't think so.... just checked. So this change should be fine I guess.